File: git_reset.py
import subprocess
import os

os.chdir('c:\\Users\\HomePC\\Desktop\\claude\\blog')

try:
    print('Resetting to commit e1ddbfae983cf0f522731cdcc3d7dc3bd6126b70...')
    result = subprocess.run(
        ['git', 'reset', '--soft', 'e1ddbfae983cf0f522731cdcc3d7dc3bd6126b70'],
        capture_output=True,
        text=True
    )
    
    if result.returncode == 0:
        print('✅ Reset successful!')
    else:
        print(f'Error: {result.stderr}')
    
    print('\n=== Current git status ===')
    status = subprocess.run(['git', 'status'], capture_output=True, text=True)
    print(status.stdout)
    
except Exception as e:
    print(f'Error: {e}')

File: git-reset.js
const { execSync } = require('child_process');
const path = require('path');

const blogDir = 'c:\\Users\\HomePC\\Desktop\\claude\\blog';
process.chdir(blogDir);

try {
  console.log('Resetting to commit e1ddbfae983cf0f522731cdcc3d7dc3bd6126b70...');
  const result = execSync('git reset --soft e1ddbfae983cf0f522731cdcc3d7dc3bd6126b70', { encoding: 'utf-8' });
  console.log('Reset successful!');
  console.log(result);
  
  console.log('\nCurrent git status:');
  const status = execSync('git status', { encoding: 'utf-8' });
  console.log(status);
} catch (error) {
  console.error('Error:', error.message);
  process.exit(1);
}
